pkg/component/nodemanagement/dependencywatchdog/access.go

```diff
@@ -89,36 +78,6 @@ func (d *dependencyWatchdogAccess) Deploy(ctx context.Context) error {
 	return d.createManagedResource(ctx)
 }
 
-// TODO(aaronfern): Remove this function after v1.93 got released.
```

Comment:
Can also https://github.com/gardener/gardener/blob/e8454f02c43d90c62fb3a7bb4c43919e8d16c971/cmd/gardenlet/app/migration.go#L66 be removed?

Reply:
I think you're looking at an outdated branch. This was removed from master in the following PR https://github.com/gardener/gardener/pull/9668

Reply:
Ah, sorry, my bad.